.circleci/config.yml

```diff
@@ -154,7 +154,29 @@ workflows:
           working_directory: sites/partners
           yarn: true
           build: |
+            echo 'export FEATURE_LISTINGS_APPROVAL=FALSE' >> "$BASH_ENV"
+            source "$BASH_ENV"
```

Comment:
Is this necessary? By default the FEATURE_LISTING_APPROVAL is not set so it would be equivalent of false. It's ok to have it here, but ideally we wouldn't touch the existing setup if we don't need to.

Reply:
For sure, the intention here was just clarity but happy to remove it!